app.js

```diff
@@ -4,40 +4,36 @@ const logger = require("morgan");
 const mongoose = require("mongoose");
 const cors = require("cors");
 
+const app = express();
+mongoose.connect(process.env.DB_URI);
```

Comment:
```suggestion
mongoose.connect(process.env.DB_URI);
```
.env 파일 변수명 통일이 필요해 보입니다.